docs/installation/deployment/kubernetes.rst

```diff
@@ -0,0 +1,280 @@
+=======================
+Deploying on Kubernetes
+=======================
+
```

Comment:
Let's indicate the knowledge level of k8s to deploy this. And add some links to external resources to get familiar with our Target platform (gcloud).

Reply:
Added a section :+1: